src/operators/test_identity.py

```diff
@@ -5,31 +5,50 @@
 Identity operators are used to compare the objects, not if they are equal, but if they are actually
 the same object, with the same memory location.
 """
-
+import copy
 
 def test_identity_operators():
     """Identity operators"""
 
     # Let's illustrate identity operators based on the following lists.
     first_fruits_list = ["apple", "banana"]
     second_fruits_list = ["apple", "banana"]
-    third_fruits_list = first_fruits_list
+    third_fruits_list = first_fruits_list   #Shallow copy
```

Comment:
Some things that are purely technically related to how the language handles it don't need comments.